cypress/tests/APIs/api-account.cy.ts

```
@@ -26,7 +26,7 @@ describe("Account API Tests", { tags: ["APIs"] }, () => {
   });
 
   context("Create/Register user account", () => {
-    it("create/register new User Account", () => {
+    it.only("create/register new User Account", () => {
```

Biome

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)